src/utils/constants.js

```diff
@@ -1,53 +1,40 @@
-export const initialCards = [
-  {
-    name: "Yosemite Valley",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/yosemite.jpg",
-  },
-  {
-    name: "Lake Louise",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/lake-louise.jpg",
-  },
-  {
-    name: "Bald Mountains",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/bald-mountains.jpg",
-  },
-  {
-    name: "Latemar",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/latemar.jpg",
-  },
-  {
-    name: "Vanoise National Park",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/vanoise.jpg",
-  },
-  {
-    name: "Lago di Braies",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/lago.jpg",
-  },
-];
-//----------------------------------------------------------------------------------------
-//                                      Elements
-//----------------------------------------------------------------------------------------
-
 const profileEditButton = document.querySelector(".profile__edit-button");
 const profileEditModal = document.querySelector("#profile-edit-modal");
+const avatarEditModal = document.querySelector("#avatar-edit-modal");
 const addCardModal = document.querySelector("#add-card-modal");
+
 const profileModalCloseButton = profileEditModal.querySelector("#modal-close");
 const addCardModalCloseButton = addCardModal.querySelector("#modal-close");
+
 const profileTitle = document.querySelector(".profile__title");
 const profileDescription = document.querySelector(".profile__description");
+
 const profileTitleInput = document.querySelector("#profile-title-name");
 const profileDescriptionInput = document.querySelector(
   "#profile-description-input"
 );
+
 const addNewCardButton = document.querySelector(".profile__add-button");
+
+//forms
 const profileEditForm = profileEditModal.querySelector("#modal-form");
+const avatarFormElement = avatarEditModal.querySelector("#modal-form");
 const addCardFormElement = addCardModal.querySelector("#modal-form");
+
+//buttons
+const profileEditSaveButton = profileEditForm.querySelector(".modal__button");
+const avatarSaveButton = avatarFormElement.querySelector(".modal__button");
+const addCardSaveButton = addCardFormElement.querySelector(".modal__button");
+
+//cards
 const cardTitleInput = addCardFormElement.querySelector("#card-title-input");
 const cardUrlInput = addCardFormElement.querySelector("#card-url-input");
 const cardListEl = document.querySelector(".cards__list");
 const cardImageEL = document.querySelector(".cards__list");
 const Template =
```

Comment:
Every variable in the project should be in `camelCase` (in **lower case** if it's not a class or component)